pydata_sphinx_theme/docs-sidebar.html

```diff
@@ -10,10 +10,14 @@
   {% set nav = get_nav_object(maxdepth=3, collapse=True) %}
 
   <ul class="nav bd-sidenav">
+      <!-- top-level pages -->
       {% for main_nav_item in nav %}
       {% if main_nav_item.active %}
+        <!-- sub-pages within each top level page -->
         {% for nav_item in main_nav_item.children %}
-          {% if nav_item.children %}
+          {% if nav_item["type"] == "caption" %}
+            <li class="nav-caption">{{ nav_item.text }}</li>
```

Comment:
Do we actually want to start a new list after each caption? That's eg how rtd does it (eg http://xarray.pydata.org/en/stable/)